vic/drivers/classic/src/write_model_state.c

```diff
@@ -146,7 +146,7 @@ write_model_state(all_vars_struct *all_vars,
                    filep->statefile);
         }
         else {
-            fprintf(filep->statefile, " %f ", soil_con->dz_node[nidx]);
+            fprintf(filep->statefile, " %.16g ", soil_con->dz_node[nidx]);
```

Comment:
suggest defining a header variable to handle this repetitive format. E.g.

```C
#define ASCII_STATE_FLOAT_FMT " %.16g "
```

Or something with a shorter but descriptive name?